kartothek/serialization/_parquet.py

```diff
@@ -22,19 +24,14 @@
     filter_df,
     filter_df_from_predicates,
 )
-from ._io_buffer import BlockBuffer
 from ._util import ensure_unicode_string_type
 
-try:
-    # Only check for BotoStore instance if boto is really installed
-    from simplekv.net.botostore import BotoStore
-
-    HAVE_BOTO = True
-except ImportError:
-    HAVE_BOTO = False
```

Comment:
Why remove this?

Reply:
the comment below mentions that there are leaked connections with this backend. did you verify this to be no longer the case?